src/angular/src/app/app.html

```
@@ -1,14 +1,14 @@
-     [class.top-sidebar-closed]="!showSidebar">
+     [class.top-sidebar-open]="showSidebar()"
+     [class.top-sidebar-closed]="!showSidebar()">
 
   <div id="logo">
     <span class="image">
       <img src="assets/logo.png" alt="" />
     </span>
     <span class="text"><b>Seed</b>Sync</span>
 
-    <button id="sidebar-btn-close" class="sidebar-btn" (click)="showSidebar=false">
+    <button id="sidebar-btn-close" class="sidebar-btn" (click)="showSidebar.set(false)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)